compiler/il/OMRIL.cpp

```diff
@@ -652,6 +653,102 @@ OMR::IL::opCodeForCorrespondingIndirectStore(TR::ILOpCodes storeOpCode)
    return TR::BadILOp;
    }
 
+TR::IL*
+OMR::IL::self()
+   {
+   return static_cast<TR::IL*>(this);
+   }
+
+TR::ILOpCodes
+OMR::IL::opCodeForCorrespondingLoadOrStore(TR::ILOpCodes opCodes)
+   {
+   TR::ILOpCode opCode(opCodes);
+
+   if (opCode.isLoadIndirect())
+      return self()->opCodeForCorrespondingIndirectLoad(opCodes);
+   else if (opCode.isLoadDirect())
+      return self()->opCodeForCorrespondingDirectLoad(opCodes);
+   else if (opCode.isStoreIndirect())
+      return self()->opCodeForCorrespondingIndirectStore(opCodes);
+   else if (opCode.isStoreDirect())
+      return self()->opCodeForCorrespondingDirectStore(opCodes);
+
+   TR_ASSERT(0, "opCode is not load or store");
+   return TR::BadILOp;
+   }
+
+TR::ILOpCodes
+OMR::IL::opCodeForCorrespondingDirectLoad(TR::ILOpCodes loadOpCode)
+   {
+   switch (loadOpCode)
+      {
+      case TR::bload: return TR::bstore;
+      case TR::sload: return TR::sstore;
+      case TR::iload: return TR::istore;
+      case TR::lload: return TR::lstore;
+      case TR::fload: return TR::fstore;
+      case TR::dload: return TR::dstore;
+      case TR::aload: return TR::astore;
+      case TR::vload: return TR::vstore;
+      case TR::cload: return TR::cstore;
+      case TR::buload: return TR::bstore;
```

Comment:
these unsigned cases are deprecated and no longer generated I believe... @0xdaryl that's right isn't it?